shells/pipes-shell-2/bus.js

```diff
@@ -8,6 +8,7 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
+import {Services} from '../../build/runtime/services.js';
```

Comment:
Here we should wait to make sure @sjmiles is ok with the changes. Can you send him a pointer to the other internal change so he can see the other side of it?

Reply:
The `bus` itself is generic, so this is not in the right place. 

The intent of the pipe-shell-2 upgrade was to force all correspondence between the two sides to happen via messages,  binding more code directly from Java is a different architecture.

Maybe this was the easiest way to get from point A to point B. We can discuss, but at the very least this shouldn't be in `bus.js`.